### PR TITLE
chore(token-import): Adding analytics for the send flow

### PR DESCRIPTION
**Pull request by dievazqu**
### Description

Adding `isTokenManuallyImported` field to the analytics in the send flow.

### CODE REVIEW

File: src/tokens/saga.test.ts [hunk 1]
```diff
@@ -381,14 +387,103 @@ describe(fetchImportedTokenBalances, () => {
...
       [mockUSDCTokenId]: {
         ...mockImportedTokens[mockUSDCTokenId],
         balance: new BigNumber(10).toFixed(),
+        priceUsd: undefined,
+      },
+    })
+  })
+})
+
+describe(fetchImportedTokenBalances, () => {
```
Comment (kathaypacific) [inline src/tokens/saga.test.ts:406]: is this a bad merge? it seems like this test group already exists?

**Review by kathaypacific — COMMENTED** [review]
i think there's a bad merge here for the tests. will there be a separate PR for the swap flow analytics? could we also make this PR title more specific to the changes (e.g. adding analytics for send flow) so that the change log is clearer? 🙏🏻

**Comment by dievazqu** [comment]
> i think there's a bad merge here for the tests. will there be a separate PR for the swap flow analytics? could we also make this PR title more specific to the changes (e.g. adding analytics for send flow) so that the change log is clearer? 🙏🏻

Yes, the analytics for the swaps are coming with the PR to improve the swap experience for the imported tokens.
And sorry for the PR title, updated.

File: src/tokens/saga.test.ts (hunk 1 above)
Comment (dievazqu) [reply src/tokens/saga.test.ts:406]: Oh, yes, I messed it up in the merge 🤯

**Review by kathaypacific — APPROVED** [review]

**Pull request merged**